### DISCUSSION TITLE
Ghosts of GitHub Past

### DISCUSSION THREAD

**Discussion by javabster** [Qiskit/.github#14] [Announcements]
## :christmas_tree: Happy holidays fellow Qiskitters! :christmas_tree:
This holiday season Father Qiskit :santa: is looking for some extra elves to help him sort out old Qiskit issues, so this week we are inviting you all to help us confront our Ghosts of GitHub Past!

Over the next 5 days we are requesting extra help on the following:
:mag: **Check if an issue is reproducible**: sometimes issues do not apply any more because, for example, the module where the bug was has been removed, or the issue is no longer reproducible with the current version of Qiskit. If this is the case let us know and we can close the issue
:speech_balloon: **Update issues with current code**: Sometimes an issue submitter says  “this does not work, here is an example”, but the example does not reproduce the issue in the current version of Qiskit. In that case, please add a comment on the issue with an updated example
🤔 : **Sanity check**: Review the issue description, is it clear what the problem is and how it can be addressed? Does it make sense? Are the labels appropriate? If not leave a comment in the issue with some clarification or let us know

### :sled: How can you get involved? :sled:

1. join the [`#ghosts_of_github_past`](https://join.slack.com/share/enQtMjgyNTY3NzM2NTY3MC1iOGMxMmYzYTBkZTU4MmE5MjA3Y2VkZTk3Y2FmNjcxOThkZGVkN2Q5YzQ4MjJkZDcxZDc4MTRmMGNjNTU1YmZj) slack channel
2. take a look at [this issue spreadsheet](https://ibm.biz/gogp_issues) to see which ones you might be interested in helping out with
3. fill in [this form](https://ibm.biz/gogp_signup) with your github username and the issue(s) you want to work on.
4. get working on your issue! Ask for help in the [`#ghosts_of_github_past`](https://join.slack.com/share/enQtMjgyNTY3NzM2NTY3MC1iOGMxMmYzYTBkZTU4MmE5MjA3Y2VkZTk3Y2FmNjcxOThkZGVkN2Q5YzQ4MjJkZDcxZDc4MTRmMGNjNTU1YmZj)  slack channel, where Father Qiskit and the other elves will be hanging out if you need help for the next 5 days
5. once you’ve addressed your issue let us know in the [`#ghosts_of_github_past`](https://join.slack.com/share/enQtMjgyNTY3NzM2NTY3MC1iOGMxMmYzYTBkZTU4MmE5MjA3Y2VkZTk3Y2FmNjcxOThkZGVkN2Q5YzQ4MjJkZDcxZDc4MTRmMGNjNTU1YmZj) slack channel so Father Qiskit will make sure to put you on his nice list! For each issue you’ve addressed leave a comment in the channel with the issue number and how you’ve addressed it, e.g:
    - :white_check_mark: “looks good”: Nothing needs to be done. The issue is good as it is.
    - :no_entry_sign: “not reproducible”: you weren’t able to reproduce the issue and it needs to be closed
    - :repeat: “now is reproducible”: Some code was added to make the issue reproducible with current Qiskit version
    - :information_source: “needs more info”: The issue is not clear and needs to be labeled as “needs more info”. Add a comment with all your questions
    - 🤔 : “moot”: issue is no longer relevant nor addressable
    - :question: “i don’t know”: the decision might be too complicated. A core maintainer needs to review it

### 🗣️ Slack communication 🗣️ 
- use the thread created by the AirTable bot
- mention `@father_qiskit` if you need help or have question for the core team
- if you commit to an issue are you don’t want to continue with it, let `@father_qiskit` know so it can get back into the pool of available issues
- if you solve the issues with step 5, let `@father_qiskit` know.

**Comment by bopardikarsoham** [comment]
The links for the issue spreadsheet and the form seem to be broken

**Reply by 1ucian0** [reply]
Thanks for the headsup! Fixed in the original message. Just for the record:
* this issue spreadsheet: https://ibm.biz/gogp_issues
* the form: https://ibm.biz/gogp_signup
